packages/ozone/src/db/migrations/20241018T205730722Z-setting.ts

```diff
@@ -0,0 +1,34 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('setting')
+    .addColumn('id', 'serial', (col) => col.primaryKey())
+    .addColumn('key', 'text', (col) => col.notNull())
+    .addColumn('did', 'text', (col) => col.notNull())
+    .addColumn('value', 'jsonb', (col) => col.notNull())
+    .addColumn('description', 'text')
+    .addColumn('createdAt', 'timestamptz', (col) =>
+      col.defaultTo(sql`now()`).notNull(),
+    )
+    .addColumn('updatedAt', 'timestamptz', (col) =>
+      col.defaultTo(sql`now()`).notNull(),
+    )
+    .addColumn('managerRole', 'text')
+    .addColumn('scope', 'text', (col) => col.notNull())
+    .addColumn('createdBy', 'text', (col) => col.notNull())
+    .addColumn('lastUpdatedBy', 'text', (col) => col.notNull())
+    .addUniqueConstraint('setting_scope_key_idx', ['key', 'scope'])
```

Comment:
I believe `(key, scope)` wont be unique when multiple users have the same `personal` setting with the same `key`.  Maybe qualifying this by `did` does the trick, and then we can nix `setting_did_idx`.